linter/lintmain/internal/check/check.go

```diff
@@ -246,6 +249,29 @@ func (l *linter) loadPlugin() error {
 }
 
 func (l *linter) parseArgs() error {
+	paramKey := func(info *lintpack.CheckerInfo, paramName string) string {
+		return "@" + info.Name + "." + paramName
```

Comment:
I love this pattern with a local func in your code

Reply:
Can't be used when a function is needed in a more than 1 method. :)

Reply:
well, yeah ;D